app/controllers/api/account.php

```diff
@@ -968,6 +969,46 @@
         $response->dynamic(new Document(['logs' => $output]), Response::MODEL_LOG_LIST);
     });
 
+App::get('/v1/account/sessions/:sessionId')
+    ->desc('Get Session By ID')
+    ->groups(['api', 'account'])
+    ->label('scope', 'account')
+    ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_JWT])
+    ->label('sdk.namespace', 'account')
+    ->label('sdk.method', 'getSessions')
+    ->label('sdk.description', '/docs/references/account/get-sessions.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_SESSION_LIST)
+    ->param('sessionId', null, new UID(), 'Session unique ID. Use the string \'current\' to get the current device session.')
+    ->inject('response')
+    ->inject('user')
+    ->inject('locale')
+    ->inject('projectDB')
+    ->action(function ($sessionId, $response, $user, $locale, $projectDB) {
+        /** @var Appwrite\Utopia\Response $response */
+        /** @var Appwrite\Database\Document $user */
+        /** @var Utopia\Locale\Locale $locale */
+
+        $sessionId = ($sessionId === 'current')
+        ? Auth::sessionVerify($user->getAttribute('sessions'), Auth::$secret)
+        : $sessionId;
+
+        $session = $projectDB->getCollectionFirst([ // Get user by email address
```

Comment:
We need to add the country name manually, since its not saved in the database.

You can find an example here: https://github.com/appwrite/appwrite/blob/master/app/controllers/api/account.php#L881-L883

Reply:
@PineappleIOnic the reason we don't have the country in the DB is that we support changing the country names to multiple languages.